client/src/components/CreateAccountForm.jsx

```diff
@@ -0,0 +1,32 @@
+import { connect } from 'react-redux';
+import { changeCreateField, doCreateAccount } from '../actions/account';
+
```

Comment:
This is the first file mentioned in my review.

Reply:
Thanks, this is a bit confusing -- I'll clean it up and rename some things.

Reply:
A bunch of the logic is duplicated and the only thing that changes are the fields...so I added an "abstract" Form component. Renamed it from `SimpleForm`  to `AbstractForm`.